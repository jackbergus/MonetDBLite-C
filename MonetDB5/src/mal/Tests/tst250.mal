# The series tst2?? is a patched version of suite_00
# The first and foremost BAT operations

b:= bat.new(:int,:int);
err := bat.insert(b,1,15);
err := bat.insert(b,2,4);
n0  := calc.int(nil);
err := bat.insert(b,3,n0);
err := bat.insert(b,4,16);
err := bat.insert(b,5,25);
err := bat.insert(b,6,36);
err := bat.insert(b,7,49);
err := bat.insert(b,8,64);
err := bat.insert(b,9,n0);
err := bat.insert(b,n0,n0);
err := bat.insert(b,n0,81);
io.print(b);
barrier (v,h,t):= chopper.newIterator(b);
	io.printf("%d,",h);
	io.printf("%d\n",t);
	redo (v,h,t):= chopper.hasMoreElements(b);
exit v;
