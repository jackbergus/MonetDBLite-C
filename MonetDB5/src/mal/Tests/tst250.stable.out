stdout of test 'tst250` in directory 'src/mal` itself:


# 08:01:03 >  
# 08:01:03 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/Linux/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB5/log" --set mapi_port=42158 --set sql_port=55708 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst250.mal </dev/null
# 08:01:03 >  

# Monet Database Server V4.99.19
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# config:/ufs/mk/monet5/Linux/etc/MonetDB5.conf
# dbfarm:/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
## The series tst2?? is a patched version of suite_00
## The first and foremost BAT operations
#
#b:= new(:int,:int);
#err := insert(b,1,15);
#err := insert(b,2,4);
#n0  := int(nil);
#err := insert(b,3,n0);
#err := insert(b,4,16);
#err := insert(b,5,25);
#err := insert(b,6,36);
#err := insert(b,7,49);
#err := insert(b,8,64);
#err := insert(b,9,n0);
#err := insert(b,n0,n0);
#err := insert(b,n0,81);
#print(b);
#barrier (v,h,t):= chopper.newIterator(b);
#	printf("%d,",h);
#	printf("%d\n",t);
#	redo (v,h,t):= chopper.hasMoreElements(b);
#exit v;
#
function user.main():void;
# The series tst2?? is a patched version of suite_00 
# The first and foremost BAT operations 
    b := bat.new(:int,:int);
    err := bat.insert(b,1,15);
    err := bat.insert(b,2,4);
    n0 := calc.int(nil);
    err := bat.insert(b,3,n0);
    err := bat.insert(b,4,16);
    err := bat.insert(b,5,25);
    err := bat.insert(b,6,36);
    err := bat.insert(b,7,49);
    err := bat.insert(b,8,64);
    err := bat.insert(b,9,n0);
    err := bat.insert(b,n0,n0);
    err := bat.insert(b,n0,81);
    io.print(b);
barrier (v,h,t) := bat.newIterator(b);
    io.printf("%d,",h);
    io.printf("%d\n",t);
    redo (v,h,t) := bat.hasMoreElements(b);
exit v;
end main;
#-----------------#
# h	tmp_4	  # name
# int	int	  # type
#-----------------#
[ 1,	  15	  ]
[ 2,	  4	  ]
[ 3,	  nil	  ]
[ 4,	  16	  ]
[ 5,	  25	  ]
[ 6,	  36	  ]
[ 7,	  49	  ]
[ 8,	  64	  ]
[ 9,	  nil	  ]
[ nil,	  nil	  ]
[ nil,	  81	  ]
1,15
2,4
3,nil
4,16
5,25
6,36
7,49
8,64
9,nil
nil,nil
nil,81

# 08:01:03 >  
# 08:01:03 >  Done.
# 08:01:03 >  

